modules/probe.sh

```diff
@@ -146,7 +146,7 @@ probe_kernel_image() {
     n=$(wc -l <<< "$kernels")
 
     if [ "$n" -gt "1" ];then
-      log_warn "Found multiple kernel images: $(awk '{print $2}' <<< $kernels)"
+      log_warn "Found multiple kernel images: $(awk '{print $2}' <<< ${kernels})"
```

Comment:
One final remark, the printout looks a bit strange,

```
2020-06-08 10:50:12 [WARN] [mender-convert-modify] Found multiple kernel images: /mender-convert/work/rootfs/boot/vmlinuz-4.15.0-20-generic
/mender-convert/work/rootfs/boot/vmlinuz
2020-06-08 10:50:12 [WARN] [mender-convert-modify] Selecting newest kernel image: /mender-convert/work/rootfs/boot/vmlinuz-4.15.0-20-generic
2020-06-08 10:50:12 [WARN] [mender-convert-modify] Set MENDER_GRUB_KERNEL_IMAGETYPE to override selected kernel
```
Would prefer it was something like this:

```
2020-06-08 10:50:12 [WARN] [mender-convert-modify] Found multiple kernel images: 

    /mender-convert/work/rootfs/boot/vmlinuz-4.15.0-20-generic
    /mender-convert/work/rootfs/boot/vmlinuz

2020-06-08 10:50:12 [WARN] [mender-convert-modify] Selecting newest kernel image: 

    /mender-convert/work/rootfs/boot/vmlinuz-4.15.0-20-generic

2020-06-08 10:50:12 [WARN] [mender-convert-modify] Set MENDER_GRUB_KERNEL_IMAGETYPE to override selected kernel
```

Same applies to initrd prints